src/sentry/integrations/slack/message_builder/disconnected.py

```diff
@@ -0,0 +1,18 @@
+from sentry.integrations.slack.message_builder import SlackBody
+from sentry.integrations.slack.message_builder.help import SlackHelpMessageBuilder
+
+DISCONNECTED_MESSAGE = (
+    "Slack has been uninstalled from your Sentry organization, re-install it to continue."
+)
+
+
+class SlackDisconnectedMessageBuilder(SlackHelpMessageBuilder):
+    def __init__(self) -> None:
+        """Override parent constructor."""
+        super().__init__()
+
+    def build(self) -> SlackBody:
+        return self._build_blocks(
```

Comment:
why use blocks here instead of just sending simple text?

Reply:
I wanted to reuse the documentation block here.

Reply:
not worth blocking this over but I think we'll be discontinuing it once we remove the feature flag, as it's been changed in the newer one: https://github.com/getsentry/sentry/pull/26983